SPECS/expat/expat.spec

```diff
@@ -55,17 +53,22 @@ rm -rf %{buildroot}/*
 %license COPYING
 %doc AUTHORS Changes
 %{_bindir}/*
-%{_mandir}/man1/*
 
 %files devel
 %{_includedir}/*
 %{_libdir}/pkgconfig/*
 %{_libdir}/libexpat.so
+%{_libdir}/cmake/expat-%{version}
 
 %files libs
 %{_libdir}/libexpat.so.*
 
 %changelog
```

Comment:
GitHub won't let me comment on the relevant lines, but a few small nits:

1) `%clean` section can be removed- this is handled automatically by RPM
2) The first line in the `%install` section which clears the buildroot can be removed- also handled by RPM
3) The `%post` and `%postun` sections can be replace by the single line `%ldconfig_scriptlets`
4) The `make %{?_smp_mflags} <target>` invocations can be replaced by `%make_build <target>`, so we get consistent verbosity/jobs usage across our specs.
5) The `make DESTDIR=%{buildroot} install` line can be replaced by `%make_install` for brevity.